docs/source/noaa_csp.rst

```diff
@@ -0,0 +1,197 @@
+     - C48
+     - ``ATM``
+     - ``c5.9xlarge (36 vCPUs, 72 GB Memory, amd64)``
+     - ``compute``
+     - ``/lustre``
+
+Instructions regarding configuring the respective CSP instance and
+cluster follows.
+
+*********************
+Login to the NOAA CSP
+*********************
+
+Log in to the `NOAA CSP <http://noaa.parallel.works/login>`_ and into
+the resources configuration. The user should arrive at the following
+screen.
+
+Note that the ``Username or email`` query is case-sensitive. The user
+will then be prompted for their respective RSA token key using the
+same application use for the other RDHPCS machines (i.e., Hera, Jet,
+etc.,).
+
+.. image:: _static/noaacsp_login.png
+
+*******************************
+Configure the NOAA CSP Instance
+*******************************
+
+Once logged into the NOAA CSP, navigate to the :red-text:`RESOURCES` section
+and click the ``+ Add Resource`` button in the upper-right corner as
+illustrated below.
+
+.. image:: _static/noaacsp_instance_1.png
+   
+Next, the mandatory attributes for the respective instance must be
+defined as shown in the illustration below.
+
+.. image:: _static/noaacsp_instance_2.png
+
+The annotated attributes and their respective descriptions are as
+follows.
+
+1. A unique name for the instance. Best practices suggest one that is
+   clear, concise, and relevant to the application.
+2. A short description of the instance, i.e., ``This instance supports
+   this <task name> task.``
+3. Tag(s) describing and identifying the respective instance. These
+   allow for improved bookkeeping, especially when a user has multiple
+   or concurrent instance types.
+
+Next, the cluster is defined as shown in the following illustration.
+
+.. image:: _static/noaacsp_instance_3.png
+
+The NOAA Parallel Works initiative currently provides 2 CSPs for the
+global-workflow; **AWS** (Amazon Web Services) and **Azure**
+(Microsoft Azure). Existing clusters may also be modified. However
+this is neither recommended or supported.
+
+Finally, when satisfied with the CSP instance configure, click ``Add
+Resource`` as illustrated below.
+
+.. image:: _static/noaacsp_instance_4.png
+
+******************************
+Configure the NOAA CSP Cluster
+******************************
+
+Navigate to the tab and locate the CSP instance configured in the
+previous section and click on the link, `globalworkflowdemo` for this
+example.
+
+.. image:: _static/noaacsp_cluster_1.png
+
+The respective CSP cluster maybe then be configured. The mandatory
+configuration attributes are as follows.
+
+- Availability zone;
+- Disk size and storage type(s);
+- Available compute and resource partitions.
+
+The following image describes the general settings for the respective
+cluster. These attributes are specific to the user and the respective
+user's group allocation. The right-most panel provides a breakdown of
+the costs related to the requested compute and storage resources. In
+addition.
```

Comment:
```suggestion
the costs related to the requested compute and storage resources. While there is a space to place an SSH key here, RDHPCS recommends you add any SSH keys to under your Account➡Authentication instead. This will allow you so connect from another machine instead of using the web terminal.
```